.github/workflows/test-code.yml

```diff
@@ -0,0 +1,51 @@
+name: "Test / Code extension"
+
+on:
+  push:
+    branches:
+      - main
+    paths: ["editors/code/**"]
+  pull_request:
+    paths: ["editors/code/**"]
+  workflow_dispatch:
```

Comment:
Nice yea I was going to request workflow-dispatch in case we change the server and want to run it manually